R/build-home-authors.R

```diff
@@ -168,6 +175,17 @@ orcid_link <- function(orcid) {
   )
 }
 
+ror_link <- function(ror) {
+  if (is.null(ror)) {
+    return(NULL)
+  }
+
+  paste0(
+    "<a href='https://ror.org/", ror, "'>",
+    "<img src='https://raw.githubusercontent.com/ror-community/ror-logos/main/ror-icon-rgb.svg' class='ror' alt='ROR'></a>"
```

Comment:
Should we use the version with transparent background?

https://github.com/ror-community/ror-logos/blob/main/ror-icon-rgb-transparent.svg

Reply:
wouldn't it look worse in dark mode?